src/sentry/api/endpoints/sentry_app_publish_request.py

```diff
@@ -41,3 +58,14 @@ def post(self, request, sentry_app):
         )
 
         return Response(status=201)
+
+    def is_issue_link_integration(self, sentry_app):
```

Comment:
This method is static so I'd make it a function outside of the class. You can also simplify:
```py
def is_issue_link_integration(sentry_app: SentryApp) -> bool:
    """Determine if the sentry app supports issue linking."""
    elements = (sentry_app.schema or {}).get("elements", [])
    return any(element.get("type") == "issue-link" for element in elements)
```